api/blocks/blocks.go

```diff
@@ -34,6 +37,10 @@ func (b *Blocks) handleGetBlock(w http.ResponseWriter, req *http.Request) error
 	if err != nil {
 		return utils.BadRequest(errors.WithMessage(err, "revision"))
 	}
+	raw := req.URL.Query().Get("raw")
```

Comment:
Should be throw a 400 if raw = true and expanded = true?

Reply:
As @libotony suggested I made the `raw` to return summary.Header, instead of summary, should I return the whole summary if `expanded = true`?

Reply:
Summary doesn't play a role of 'expanded' block, it is designed for internal storage structure.